extensions/amp-carousel/amp-carousel.md

```diff
@@ -153,6 +162,16 @@ corresponding carousel item displays.
 
 [/example]
 
+### Accessibility considerations for `amp-carousel`
+
+Autoplaying, and particularly infinitely looping, carousels can be very distracting and confusing for users - especially for users with cognitive impairments. In general, we recommend avoiding autoplaying carousels. While autoplaying carousels stop once the user has interacted with the carousel, consider consider also adding an explicit "Play/Pause" control.
+
+Depending on the content of the slides, it's possible that using a `delay` that is below ~350ms can result in strobing effects that, in extreme cases, can trigger seizures in users with photosensitive seizure disorders. As a general rule, never use `delay` values below `1000`.
```

Comment:
By default the autoadvance interval is 5000ms and cannot be below `1000`. This statement can be removed entirely or updated to explain why the interval is a minimum of `1000`.

Reply:
It seems the minimum value of `1000` was not mentioned. Removed this statement, and added mention of the minimum to the explanation for `delay` instead.

Reply:
Thank you!